### ISSUE TITLE
ado

### ISSUE THREAD

**Issue by jamesdbrock** [purescript/documentation#427]
I think the *Syntax* page should explain `ado`?

https://github.com/purescript/documentation/blob/52282671d755a37f9acaaaa8570c760487dd920b/language/Syntax.md

**Comment by JordanMartinez** [comment]
Yeah, as it's own separate section following the Do Notation one: https://github.com/purescript/documentation/blob/52282671d755a37f9acaaaa8570c760487dd920b/language/Syntax.md#do-notation=

**Comment by jamesdbrock** [comment]
Also qualified do.
